tools/secrets-tool/commandhelper.cpp

```diff
@@ -65,6 +65,39 @@ static Sailfish::Crypto::CryptoManager::DigestFunction digestEnum(const QString
     return Sailfish::Crypto::CryptoManager::DigestSha512;
 }
 
+static QVariantMap toCustomParameters(const QStringList &options)
+{
+    QVariantMap customs;
+    for (QList<QString>::ConstIterator it = options.constBegin(); it != options.constEnd(); it++) {
+        QStringList keyValue(it->split('='));
+        if (keyValue.length() == 2
+            && !keyValue[0].isEmpty() && !keyValue[1].isEmpty()) {
+            const QString key = keyValue.takeFirst();
+            const QString value = keyValue.takeFirst();
+            bool ok;
+            QVariant vd(value.toDouble(&ok));
+            if (ok) {
+                customs.insert(key, vd);
+            } else {
+                QVariant vi(value.toInt(&ok));
+                if (ok) {
+                    customs.insert(key, vi);
+                } else {
+                    if (value.compare("true", Qt::CaseInsensitive) == 0)
```

Comment:
Sailfish OS coding style is to use brackets even around single-line conditional blocks.

Reply:
Ah, ok, I was using the same convention than for sailfish-office (which is following the one of Qt), advicing not to put brackets if all conditions branches fit on one line.

I've updated the PR.